Dockerfile

```diff
@@ -1,12 +1,12 @@
-FROM python:2.7-slim
+FROM python:3.6-slim-jessie
```

Comment:
Any reason you chose the `-slim-jessie` image over the `-slim` or just the regular `python:3.6` image?  I'm wondering if some of the linux packages you are installing below might already be included in one of the other python images.  Might speed up the build time a bit.

Reply:
Using the slim base cuts this docker image size in half.

Reply:
Right -- I usually use the `slim` versions also unless I have some issue installing things on the container.  Makes sense then re: openssl version.